packages/theme-chalk/src/button.scss

```diff
@@ -38,7 +38,6 @@ $button-icon-span-gap: map.merge(
   text-align: center;
   box-sizing: border-box;
   outline: none;
-  margin: 0;
```

Comment:
Browser user agent default is margin 0. (Ant design doesn't have it either.)
If we have it, we can not use `mt-2` to change default margin.